Source/JavaScriptCore/inspector/agents/InspectorDebuggerAgent.cpp

```diff
@@ -661,6 +661,78 @@ Protocol::ErrorStringOr<void> InspectorDebuggerAgent::removeBreakpoint(const Pro
     return { };
 }
 
+static String getFunctionName(JSC::CodeBlock* codeBlock)
+{
+    if (!codeBlock)
+        return nullString();
+
+    if (codeBlock->codeType() != JSC::CodeType::FunctionCode)
+        return nullString();
+
+    return JSC::jsCast<JSC::FunctionExecutable*>(codeBlock->ownerExecutable())->ecmaName().string();
+}
+
+Protocol::ErrorStringOr<void> InspectorDebuggerAgent::addSymbolicBreakpoint(const String& symbol, std::optional<bool>&& caseSensitive, std::optional<bool>&& isRegex, RefPtr<JSON::Object>&& options)
```

Comment:
Why are these optionals passed in as rvalues?

Reply:
that's just how the Web Inspector protocol does it 😅